requirements/ingest/clarifai.in

```diff
@@ -0,0 +1,3 @@
+-c ../constraints.in
+-c ../base.txt
+clarifai==10.0.1
```

Comment:
I assume the pin is to avert similar potential hardships we've encountered with other providers on version changes before. I'd still like to advocate for not putting pins if there's no particular reason for that specific connector. 

In my defense I've checked all of our ingest requirements and only found [pinecone](https://github.com/Unstructured-IO/unstructured/blob/main/requirements/ingest/pinecone.in) and [partly github](https://github.com/Unstructured-IO/unstructured/blob/main/requirements/ingest/github.in) to be version restrictred. None of the other providers seem to be creating problems with version changes for now; and we still enjoy the updates / dependency resolution flexibility that comes with their versions being unrestricted